models/servicenow__ticket_enhanced.sql

```diff
@@ -15,23 +15,23 @@ problem as (
   select *
   from {{ ref('stg_servicenow__problem') }}
 ),
--- actually, because the relationship between incident and problem is many to many, it wouldn't make sense to left join Incident > Problem > Task. Therefore Incident would need to be its own model. So we won't use incident table in the Task Enhanced Mode. But we will use stg_sn__incident to create an aggregated model on the Problem grain in order to left join metrics like sum(incidents) per problem
-
-incident as (
-    
-  select *
-  from {{ ref('stg_servicenow__incident') }}
-),
-
-incidents_per_problem as (
-
-  select
-    problem_id_value,
-    count(distinct incident_id) as total_incidents
-    -- ideally would have fields such as 'total_high_severity_incidents' and 'total_medium_severity_incidents' that would tell you incidents per problem by severity. But severity level is customizable so we may be limited here
-  from incident
-  group by 1
-),
+-- actually, because the relationship between incident and problem is many to many, it wouldn't make sense to left join Incident > Problem > Task. Therefore Incident would need to be its own model. So we won't use incident table in the Task Enhanced Mode. But we will use stg_sn__incident to create an aggregated model on the Problem grain in order to left join metrics like sum(incidents) per problem 
+ 
+incident as ( 
+     
+  select * 
+  from {{ ref('stg_servicenow__incident') }} 
+), 
+ 
+incidents_per_problem as ( 
+ 
+  select 
+    problem_id_value, 
+    count(distinct incident_id) as total_incidents 
+    -- ideally would have fields such as 'total_high_severity_incidents' and 'total_medium_severity_incidents' that would tell you incidents per problem by severity. But severity level is customizable so we may be limited here 
+  from incident 
+  group by 1 
+), 
```

Comment:
good idea, will do this

Reply:
I actually realized the measure I was trying to grab from the incident object (total incidents per problem) is already a field in the problem object (related_incidents). Verified this in hex. As such I'll remove this CTE since now there's no use for it